src/home/homework-luks.c

```diff
@@ -1,5 +1,6 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 
+#include <libfdisk.h>
```

Comment:
we so far don't include this directly but only via fdisk-util.h. no need to change that...